README.txt

```diff
@@ -217,7 +217,7 @@ Options
 
    Files, directories, function names, and class names that match this
    regular expression are considered tests.  Default:
-   (?:^|[b_./-])[Tt]est [NOSE_TESTMATCH]
+   (?:\b|_)[Tt]est [NOSE_TESTMATCH]
```

Comment:
This helps:

```diff
diff --git a/nose/config.py b/nose/config.py
index 5997aaf..7609f35 100644
--- a/nose/config.py
+++ b/nose/config.py
@@ -499,7 +499,8 @@ class Config(object):
             dest="testMatch", metavar="REGEX",
             help="Files, directories, function names, and class names "
             "that match this regular expression are considered tests.  "
-            "Default: %s [NOSE_TESTMATCH]" % self.testMatchPat,
+            "Default: %s [NOSE_TESTMATCH]" % (
+                self.testMatchPat.replace('\\', '\\\\'),),
             default=self.testMatchPat)
         parser.add_option(
             "--tests", action="store", dest="testNames", default=None,
```

It means that you see a double backslash in `nosetests -h` though--but I don't think it's worth worrying about... it communicates what it needs to.